src/test/java/uk/gov/companieshouse/chd/order/api/ChdOrderApiApplicationTests.java

```diff
@@ -1,13 +1,195 @@
 package uk.gov.companieshouse.chd.order.api;
 
+import org.junit.Rule;
+import org.junit.contrib.java.lang.system.EnvironmentVariables;
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 class ChdOrderApiApplicationTests {
 
-	@Test
-	void contextLoads() {
-	}
+    @Rule
+    public EnvironmentVariables environmentVariables = new EnvironmentVariables();
+    private static final String CHS_API_KEY = "CHS_API_KEY";
+    private static final String CHPRD_DATASOURCE_URL = "CHPRD_DATASOURCE_URL";
+    private static final String CHPRD_SCHEMA_NAME = "CHPRD_SCHEMA_NAME";
+    private static final String CHPRD_PASSWORD = "CHPRD_PASSWORD";
+    private static final String CHPRD_CUSTOMER_ID = "CHPRD_CUSTOMER_ID";
+    private static final String CHPRD_PAYMENT_METHOD = "CHPRD_PAYMENT_METHOD";
+    private static final String CHPRD_HANDCSR = "CHPRD_HANDCSR";
+    private static final String CHPRD_STATUS = "CHPRD_STATUS";
+    private static final String CHPRD_FLAGS = "CHPRD_FLAGS";
+    private static final String CHPRD_LANGUAGE = "CHPRD_LANGUAGE";
+    private static final String CHPRD_DELIVMETH = "CHPRD_DELIVMETH";
+    private static final String CHPRD_DELIVLOCATION = "CHPRD_DELIVLOCATION";
+
+    @Test
+    void checkEnvironmentVariablesAllPresentReturnsTrue() {
+        setAllEnvironmentVariables();
```

Comment:
You could probably put this in a @BeforeEach

Reply:
Good shout 👍